testing/MLDB-1003-s3-load-dataset.js

```diff
@@ -17,7 +17,7 @@ function assertEqual(expr, val, msg)
 csv_conf = {
     type: "import.text",
     params: {
-        dataFileUrl : "s3://benchm-ml--main/test.csv",
+        dataFileUrl : "s3://dev.mldb.datacratic.com/testing/MLDB-1003/test.csv",
```

Comment:
OK.  I know what is happening.  Because the test fail once with my credentials and that I could not list the bucket, I concluded the I could not access that resource.  Now that I know better, thanks to Wolfgang, and understand that listing and accessing are two different beasts, I try the test and it worked.  I will remove that change.  MLDBFB-484 still remain open because there is at least one more public test that relies on private resources.